snap/snapcraft.yaml

```diff
@@ -57,6 +57,16 @@ parts:
       - -usr/share/discord/chrome-sandbox
       - -usr/bin/xdg-open
 
+  # Needed for setting the proper desktop file name in the electron apps
+  patch-desktop-file-name:
+    after: [discord]
+    plugin: nil
+    build-snaps: [astral-uv]
+    override-build: |
+      uv run \
+        https://raw.githubusercontent.com/snapcrafters/patch-desktop-file-name/refs/heads/uv-standalone/electron/patch-desktop-filename.py \
```

Comment:
If the PR adding this gets merged, we'd change this URL to point to the `main` branch

Reply:
Thanks a lot. Let me review it.